tools/ci_connector_ops/ci_connector_ops/utils.py

```diff
@@ -74,6 +75,23 @@ def get_changed_acceptance_test_config(diff_regex: Optional[str] = None) -> Set[
     return {Connector(get_connector_name_from_path(changed_file)) for changed_file in changed_acceptance_test_config_paths}
 
 
+def get_gradle_project_dependencies_paths(gradle_file_path: Path):
+    dependencies_paths = []
+    with open(gradle_file_path) as gradle_file:
+        for line in gradle_file:
+            if "implementation project" in line or "implementation files" in line:
+                match = re.search(GRADLE_PROJECT_RE_PATTERN, line)
+                if match:
+                    project_name = match.group(2)
```

Comment:
Given that were doing something a little custom and regexs are involved I would advocate that we add comments with very explicit example of 

We are looking for a line like
```java
    implementation project(':airbyte-integrations:bases:base-java')
```

and want to extract `airbyte-integrations:bases:base-java`